besser/bot/nlp/__init__.py

```diff
@@ -85,3 +85,27 @@
 
 default value: ``None``
 """
+
+NLP_STT_SR_ENGINE = Property(SECTION_NLP, 'nlp.speech2text.sr.engine', str, None)
+"""
+The name of the transcription engine for the Speech Recognition bot component. If none is provided, the component will not be 
+activated.
+
+name: ``nlp.speech2text.sr.engine``
+
+type: ``str``
+
+default value: ``None``
+"""
+
+NLP_STT_SR_LANGUAGE = Property(SECTION_NLP, 'nlp.speech2text.sr.engine.language', str, None)
```

Comment:
Maybe we can use the NLP_LANGUAGE property instead of creating a separate one for the speech2text language, it makes more sense to me to have a single language property for all the bot components

Reply:
This was added because I thought the language parameter for the SR lib expected a different format than the one we generally use. Yet after testing, I found out that this is not case. So it's removed now.